doc/arkode/guide/source/Usage/index.rst

```diff
@@ -78,4 +80,6 @@ ARKBBDPRE can only be used with NVECTOR_PARALLEL.
    ARKStep/index.rst
    ERKStep/index.rst
    SPRKStep/index.rst
+   SplittingStep/index.rst
+   ForcingStep/index.rst
    MRIStep/index.rst
```

Comment:
Is there some logic behind inserting the new steppers in the middle of this list?  In the absence of some clear justification, I think that this should retain the chronological ordering, with the new steppers put at the end of the list, and MRIStep before SPRKStep.